packages/discord.js/src/managers/UserManager.js

```diff
@@ -95,16 +95,6 @@ class UserManager extends CachedManager {
     return this._add(data, cache);
   }
 
-  /**
-   * Sends a message to a user.
-   * @param {UserResolvable} user The UserResolvable to identify
-   * @param {string|MessagePayload|MessageCreateOptions} options The options to provide
-   * @returns {Promise<Message>}
-   */
-  async send(user, options) {
```

Comment:
I'm...torn, it doesn't hurt to have this but I understand why we'd also remove it. 🤷 

@discordjs/core do we keep it?

Reply:
It's not a method for users. It should still be on the guild member and user class as a helper, though.

Reply:
Naw, lets keep it at the manager level, otherwise why have we made `client.channels.sendMessage`